pages/api/submit.tsx

```diff
@@ -181,4 +183,28 @@ const processFormData = async (
   }
 };
 
+/**
+ * Push a given file to a temporairy S3
+ * @param fileOrArray
+ * @param reqFields
+ * @param key
+ */
+const pushFileToS3 = async (
+  fileOrArray: formidable.File,
```

Comment:
Could be renamed to just `file` to avoid confusion as through the logic tests earlier in the parent function the array type is already test for.  The type on this input parameter is also `formidable.File` so if Typescript detected that an array could be passed it would throw an error/warning.